cmd/app/serve.go

```diff
@@ -255,6 +271,22 @@ func runServeCmd(cmd *cobra.Command, args []string) {
 		}
 	}
 
+	if viper.GetString("port") == viper.GetString("grpc-port") {
+		p := viper.GetString("port")
+		port, err := strconv.Atoi(p)
+		if err != nil {
+			log.Logger.Fatal("%s is not a valid port", p)
+		}
+		mp := viper.GetString("metrics-port")
+		metricsPort, err := strconv.Atoi(mp)
+		if err != nil {
+			log.Logger.Fatal("%s is not a valid port", mp)
+		}
+		if err := StartDuplexServer(ctx, cfg, ctClient, baseca, port, metricsPort); err != nil {
+			log.Logger.Fatal(err)
+		}
```

Comment:
I think we're missing a `return` statement after this, since we wouldn't want to go on and restart everything again